carl/envs/rna/rna_environment.py

```diff
@@ -0,0 +1,352 @@
+"""
+Code adapted from https://github.com/automl/learna
+"""
+
+import time
+
+from itertools import product
+from dataclasses import dataclass
+from distance import hamming
+
+import numpy as np
+
+from RNA import fold
+import gym
+
+
+@dataclass
+class RnaDesignEnvironmentConfig:
+    """
+    Dataclass for the configuration of the environment.
```

Comment:
Minor: Docstrings would be better readable with one blank line after each section